src/azure-cli/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -347,8 +353,10 @@ class CLIKeyTypeForBYOKImport(str, Enum):
                    help='Elliptic curve name. For valid values, see: https://docs.microsoft.com/en-us/rest/api/keyvault/createkey/createkey#jsonwebkeycurvename')
 
     with self.argument_context('keyvault key import') as c:
-        c.argument('kty', arg_type=get_enum_type(CLIKeyTypeForBYOKImport),
+        c.argument('kty', arg_type=get_enum_type(CLIKeyTypeForBYOKImport), validator=validate_key_import_type,
                    help='The type of key to import (only for BYOK).')
+        c.argument('curve', arg_type=get_enum_type(CLIJsonWebKeyCurveName), validator=validate_key_import_type,
```

Comment:
how about `crv` to be consistent with Json payload?

Reply:
Using `curve` here is for the consistentency with existing command `az keyvault key create`, where it uses `--curve` https://github.com/Azure/azure-cli/pull/16593/files#diff-cb5cdc067256b2d2d12c62d95973393cdf48d1105d65833453fab39cefc01fffL346